test/fixtures/OneDrive-Integration-Tests_690026202/Retrieves-Document-from-a-private-repository-with-credentials_809520207/recording.har

```diff
@@ -0,0 +1,361 @@
AaAaaaa0AaAaaaAaaaAAAaAaAaAAaaaAAaAaAaaAAAaAAaaAAaAaAaAAAaA0a0AAAaAaAaAAaaaAAaAaAaaAAAa0AaAaAaAAAaAAAaaAAaAaAAAAA0aAa0AaAaAA0Aa0aaA0aaaaAaAAAaAaa0AA0aAAA0AAA0AaaaaaAAAaaaaAAaAA00A0AaA0aaaa0aA00aAAA0AaaAAaaaaAaaAaaaAaA0AaAaAAAaA0AaAA00Aaa0AAa0AAAaAAAaAAa0AAAaAAAaAaaaaA0aaAAaA00aaAAaAaAaAAaaaAAaAAAaa0AaAaAaaAAaAaAaaaA0AaaaaAa0A0AAAAAaAaAaA00aAaaaaAAaAaaaaAAAaAaAA0AAAAaaA0aAAaaaaa0AAAAaAaAaaaA0AaAaAAAaAaaaAAAaAaaaAaa0AaAaAAAaA0AaAA00Aaa0AAaaAAAaAaAaAAa0AAa0AAA0Aa0aAaaaa000A0A0AaaaAaAaAAAaAaA0aA.aaAaAaA0a0aAAAa_0A00A-AaaA0aAAaA0AAaA0AaaaA0AaAaaaAAAaAAAaA0AaaAAAAaaaA0a0aaa0AAa0aaAAaaaAAAAA0a0-Aaa0AaaaAaAAaaAa0aa-0AaaA0AaA0aaAa0a0000aaAaaAaaAaAaaAaAAaAaAA00AaaaaAAaaa0a0A000AaaaAaAAAAAaAAAaAaaAAA00a0AaAA00aA0a00Aa0A0AAAa0AaaaAAaaAaaAaaAAaAa0aaAaAaaaaaaaAAAaAaaAAAaaaAAaAAAAaAAAAaaAAaaaAaaAaaaAAaaA0a0aAaA_AA0aAaaAaaAAaaa0a-aaaAaAAA00Aaa\",\"refresh_token\":\"0.AAAAAaa0-aA0a0AAaaAAaA0A0AAaa0AAA00AaaAA0A0AAAAaAAa.AaAAAAAAAAA--AAA0AA0AaaaaAa0AaaaAaAa_aAA0A-aAaaaAAaaAAa_0aaaAaA0AA0aaaaa0a0A0aaa_aaA_0a0AaaaAAaaAAA00aaaAaA-AAAaAAAAaAaAAa0aaaAaaAaAaaAAAa00A0aAaAAaAaAaA0Aa0AaaaAAa0-0aAAaaaAaaA-0aAAaA0AAAaaaaaaaAAaAAAaAAaAAaAaA00aAAA0AaAaAaaAA0-aA-aa0aaAAAaaaaAaaaAaa0a0AaaaaA0AAaAAaAaAAaAAAaaaAa000aa0aAAA0aAA0Aa0AaaaAA-AaAaAAA0aa00aAa-0aAA0AaAaaAaaAaAAaAaaAAAaAAaA0-aAaA0A00AaA-A_aAaaAaaaaaAAaAAaa0AAA0aaaAaAAaAAaaAAAAa0aAA0AAAaaAAAaaaAAa00aA0aa0aAAa0_AaAaA0AaA_aAAA_aAaaA00aaA0aaAAaaa0aAaA0aaAA0AaAAAaaAaa0aaaAAaaAaAAAA0aa00aAAaa0A0-aaAAAAaAa00Aaa-0AaAaa0aaaaaA-AA0aaaaaaAaAAaAaAAAAAaAaaAaAaA0aaaAa0a0aaAaaaA0AAa0aAaaAa0Aaaaa0aa0AAaaaAAAa_a0AaaaaAAa0aAAAa0aaAaAA_AAAaA0Aa0aA0Aa0aaaaaaAAA0A0AAAaa0AaaAaAAAaAaa0000aaaAaAAaAa0aAAaaaA0aaAaA0aaaaAAaAAaAAaaA00a0AaAAaa0A0aA_aAAa0aa0aaaAAAAAaAAaAaaAaA0aAA0AA0AA0aaaAAA0aAaaa0aAaAAaAA0aAAaaaA0aaa0AAA0aaAa0Aa00a00AaAAaA-Aaa0aaAaAAA000aAaaA0aa0AAAaAa0aa_0a_0AaA-AaAA-aA_AAaAaaAAAaa0AAaaaa0AAAAa0AaA-AA00AAAaA_0AA0AAAaa0a0a_A-A0AaaaaAa0AA0aAaaaA0Aaa0AA0AaAaa0AaAaAAaAAaaaaAAA_0aa0AAaaaAaaa0aaA-AaAAaAAaA00AAa_Aa-aA0aaAA0A00aaaaaaaaAAA0AaAAAA0AaAAA0a0AAaaAaAaAa0aAa0A0aAa0A00AaAAAAaAAaaAAAAaA_Aaa0aAAAAaAaA0A0aAAA_a0a0a0AaaAAaA000AAaaa-AaaAaaaAAa0a0Aa00\",\"id_token\":\"aaA0aAAaAaAAA0AaAAAaaAaaAaAaa00aAa0.aaAaaAAaAaA0A0AaAaaaAa00AaAaAAAaAAAaAAAaAa0aAAAaAaAaAAA0AAAaAAAaa0AaAaAaaAAaaaaaA0A0aa00aA0aa0aaAa0aaA0aAAaaAAA0AA00AaA0AAAaAAaaAAAaAA0aAaAaAaaaAAAaAAAaAaaaaAA0AaaaAaA0AAA0AaA0AAAaAaAaAaA0AAa0AaA0AaaaAaA0aAA0AAAaAAA0AAAaAAaaAA0aAaaaAaA0AAAaAa0aAAAaAAAaAA0aaAaaaaAaAAA0AaaaaAa0AAAaaAAaaAAaAAAaaAAaaa0aAA0aAaaaAAAaaaAaaAAaAaaaAAAaaaA0AaAaAa00Aa0aAAAaAAaaAAAaAA0aAaaaAAAaaaAaaAAAAAaaaAAAaaa0AAAaAaaaa0aaAaaaAaaaAAA0AaAaAaaaAa00AAAaAAaaAAaaAAAaA0A0AAAaAAaaAaaaa00aaaAaA0AaAAA0AaAaAA00AAAaAAa0Aaa0AAAaAA0aAAAaAAa0Aaa0AAaaAAaaAAAaAA00AaA0AaAaAAAaaAA0AaAaAAAAAAaaaAaaaaA0aAAAAAA0AAAaAaaAAAAAaAA0AAAAAAAAaAAAAaAAAAAAAaaaAAaaAaAaAaA0AaA0AaAAAAaAA0AaAa0aaA0aAAAAA0AaaaAaAAAAAAAaa0AAAAa0Aa0AAAAAA00aAAA0aAAaAaAaAAaaAAA0AA00AaA0AAAaAAaaAAAaAA0aAaAaAaaaAAAaAAAaAAA0aaaaaAAaaaAaAAA0AaaaaAa0A0AAAAAaAaAaA00aAaaaaAAaAaaaaAAaaAaaA0AaAA0aAA0aa00aAAA0AAAaAaAaAaAaaA.\"}"
```

Comment:
I trust that the tokens are as fake as they look.

Reply:
:-) it's your `scramble`  function... but I spent quite some time looking at the hars to ensure that nothing is leaked. I'm 99% sure. maybe not so much with the soap responses from okta...